params/config.go

```diff
@@ -292,20 +292,28 @@ type NodeConfig struct {
 	// discoverable peers with the discovery limits.
 	RequireTopics map[discv5.Topic]Limits `json:"RequireTopics"`
 
-	// StatusServiceEnabled enables status service api
-	StatusServiceEnabled bool
-
-	// InstallationId id of the current installation
-	InstallationID string
-
-	// DebugAPIEnabled enables debug api
-	DebugAPIEnabled bool
-
 	// MailServerRegistryAddress is the MailServerRegistry contract address
 	MailServerRegistryAddress string
+}
 
+// StatusServiceConfig defines options used by shhext service.
+type StatusServiceConfig struct {
+	// Enabled should be true if we want to use status service.
+	Enabled bool
+	// BackupDisabledDataDir is the file system folder the node should use for any data storage needs that it doesn't want backed up.
+	BackupDisabledDataDir string `validate:"required"`
+	// InstallationId id of the current installation
+	InstallationID string
```

Comment:
Are you sure PFS related stuff should be here? It looks like `StatusService` encapsulate options for `services/status` and `services/sshext`. Unless there is an idea to merge them.

Reply:
it looks like it makes sense to keep a single config cause all PFS related functionality is kept under same shhext namespace. So splitting config into two would cause more confusion